commands/device/list-applications.ts

```diff
@@ -0,0 +1,20 @@
+///<reference path="../../../.d.ts"/>
+"use strict";
+
+import options = require("./../../options");
+
+export class ListApplicationsCommand implements ICommand {
+	constructor(private $devicesServices: Mobile.IDevicesServices) { }
+
+	allowedParameters: ICommandParameter[] = [];
+
+	public execute(args: string[]): IFuture<void> {
+		return (() => {
+			this.$devicesServices.initialize(undefined, options.device, {skipInferPlatform: true}).wait();
```

Comment:
It seems that the first parameter is not always required, and the second one is declared as optional (```initialize(platform: string, deviceOption?: string, options?: IDevicesServicesInitializationOptions): IFuture<void>;```)

I think we should move **all** parameters into the IDevicesServicesInitializationOptions struct. Opinions?